website/content/en/preview/concepts/scheduling.md

```diff
@@ -137,6 +141,7 @@ Take care to ensure the label domains are correct. A well known label like `karp
 | -------------------------------------------------------------- | ----------  | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | topology.kubernetes.io/zone                                    | us-east-2a  | Zones are defined by your cloud provider ([aws](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html))                     |
 | node.kubernetes.io/instance-type                               | g4dn.8xlarge| Instance types are defined by your cloud provider ([aws](https://aws.amazon.com/ec2/instance-types/))                                                           |
+| node.kubernetes.io/windows-build                               | 10.0.17763  | Windows OS build in the format "MajorVersion.MinorVersion.BuildNumber". Can be `10.0.17763` for WS2019, or `10.0.20348` for WS2022. ([k8s](https://kubernetes.io/docs/reference/labels-annotations-taints/#nodekubernetesiowindows-build)) |
```

Comment:
Is this saying that Karpenter has baked-in knowledge of which AMI to look up for those two specific strings?

Fine if that's the case; if not, we should clarify.

Reply:
It does know this yes. If you were to have both AMIFamilies available on your cluster, it would know which one to pick based on this label

Reply:
Thanks @jonathan-innis for helping answer. @sftim Please resolve the conversation unless you have further comments regarding this.